packages/hamburger-react/src/Hamburger.tsx

```diff
@@ -20,15 +31,33 @@ export const Hamburger = ({ isOpen, onClick, inverted = false, description = "Ho
         className,
     );
 
+    const handleClick = () => {
+        setCanAnimate(true);
+        onClick();
+    };
+
     return (
         <button
             type="button"
             aria-label={description}
-            onClick={onClick}
+            onClick={handleClick}
             className={componentClassname}
             data-testid="jkl-hamburger"
         >
             <span className="jkl-hamburger__inner"></span>
+            {openLabel && closeLabel && (
```

Comment:
hvordan skal vi vite at du må ha både openLabel og closeLabel for at de skal vise? Er det muligens bedre å ta inn et optional objekt i props som inneholder openLabel og closeLabel?

Reply:
Hmmja, det er et godt poeng! Objekt er nok en ok løsning. Ikke helt glad i det i props, men et litt unikt case at man må ha begge deler.